### PR TITLE
merge branch: feature-std-dev with main

### PR DESCRIPTION
**Pull request by elena-rae**
ready to merge?

### CODE REVIEW

**Comment by marinalexs** [comment]
Jaaa! Lass uns mergen und dann die Merge-Konflikte lösen.

File: inflammation/models.py [hunk 1]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (marinalexs) [inline inflammation/models.py]: Could have used np.std to compute standard deviation

File: inflammation/models.py [hunk 2]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
```
Comment (marinalexs) [inline inflammation/models.py:47]: This function actually returns a dicct and not the standard deviation

**Review by marinalexs — COMMENTED** [review]
Nice work, check my comments!

File: inflammation/models.py (hunk 1 above)
Comment (elena-rae) [reply inflammation/models.py]: Ahh that's so much more efective! Lets do this!

File: inflammation/models.py (hunk 2 above)
Comment (elena-rae) [reply inflammation/models.py:47]: well spotted, will change it

Comment (marinalexs) [reply inflammation/models.py:47]: good change!

**Review by marinalexs — APPROVED** [review]
These changes look good.

File: inflammation/models.py (hunk 1 above)
Comment (elena-rae) [reply inflammation/models.py]: function adapted, see commit [914e1ac](https://github.com/elena-rae/python-intermediate-inflammation/commit/914e1ac4ccfc8cf4fbc7387172557a0697bfc7c1)

File: inflammation/models.py (hunk 2 above)
Comment (elena-rae) [reply inflammation/models.py:47]: function adapted, see commit [914e1ac](https://github.com/elena-rae/python-intermediate-inflammation/commit/914e1ac4ccfc8cf4fbc7387172557a0697bfc7c1)

**Pull request merged**